packages/auth-providers-api/src/dbAuth/shared.ts

```diff
@@ -7,18 +7,16 @@ import * as DbAuthError from './errors'
 // names.
 // Checks for cookie in headers in dev when user has generated graphiql headers
 export const extractCookie = (event: APIGatewayProxyEvent) => {
-  let cookieFromGraphiqlHeader
   if (process.env.NODE_ENV === 'development') {
     try {
-      cookieFromGraphiqlHeader = JSON.parse(event.body ?? '{}').extensions
+      const cookieFromGraphiqlHeader = JSON.parse(event.body ?? '{}').extensions
         ?.headers?.cookie
+      return cookieFromGraphiqlHeader
```

Comment:
Won't this just always return `undefined` if no GraphiQL headers are present? Since the check is using `?.` it won't ever trigger the `catch` block and so always return `undefined`, in which case the `event.headers.cookie` return will never happen? 

The previous logic seemed correct, just backwards: maybe it should try returning `cookieFromGraphiqlHeader` first, then fall back to `event.headers.cookie`?